tox.ini

```diff
@@ -1,18 +1,17 @@
 [tox]
-envlist = py27, py35, py36, py37, py38, py39, pep8, pypy
+envlist = py36, py37, py38, py39, pep8, pypy3
 
 [testenv]
 usedevelop = True
 sitepackages = False
 deps =
     .[doc]
     pytest
-    typeguard;python_version>='3.0'
+    typeguard
 commands =
-    py{27,py}: pytest --ignore='tenacity/tests/test_asyncio.py' {posargs}
-    py3{5,6,7,8,9}: pytest {posargs}
-    py3{5,6,7,8,9}: sphinx-build -a -E -W -b doctest doc/source doc/build
-    py3{5,6,7,8,9}: sphinx-build -a -E -W -b html doc/source doc/build
+    py3{6,7,8,9,py3}: pytest {posargs}
```

Comment:
This ends up being `py3py3` I don't think it'll work :)

Reply:
Ah good catch, thanks! Fixed.